### DISCUSSION TITLE
Pending transaction

### DISCUSSION THREAD

**Discussion by mojtabatouzandeh** [blockscout/blockscout#9759] [Support]
I made some Dijon Bridge almost an hour ago, it was deducted from my account, but the Dijon was not deposited for me. please follow up

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
